CharlesProxy/CharlesProxy.munki.recipe

```diff
@@ -26,6 +26,13 @@
             <string>%NAME%</string>
             <key>unattended_install</key>
             <true/>
+            <key>postinstall_script</key>
+            <string>#!/bin/sh
+
+echo "Setting permissions on Charles.app..."
+chmod -R g+w "$3/Applications/Charles.app/Contents/MacOS/Charles Proxy Settings"
```

Comment:
What is $3 for? This is a Munki postinstall_script, not a package postinstall script. $3 has no meaning in this context.
And since we want to set owner and SUID bit:

```
/usr/sbin/chown root:admin "/Applications/Charles.app/Contents/MacOS/Charles Proxy Settings"
/bin/chmod u+s "/Applications/Charles.app/Contents/MacOS/Charles Proxy Settings"
```

(or similar)